packages/stream_chat/lib/src/client/client.dart

```diff
@@ -1995,6 +2045,10 @@ class ClientState {
     channels = channels..remove(channelCid);
   }
 
+  set blockedUserIds(List<String> blockedUserIds) {
+    currentUser = currentUser?.copyWith(blockedUserIds: blockedUserIds);
+  }
```

Comment:
Should this be a public setter? I don't think it makes a lot of sense to set this from somewhere, because setting it on a client feels to me like it should be send to the backend.

Reply:
It's part of the `ClientState` so shouldn't be a problem

Reply:
ClientState is exposed to the public world, so adding it here makes it part of our public API. If we don't want or expect other devs to use this method we should just make it private. Everything we make public from this package is public api.

If it's public for testing we should just mark it with `visibleForTesting`

```suggestion
  @visibleForTesting
  set blockedUserIds(List<String> blockedUserIds) {
    currentUser = currentUser?.copyWith(blockedUserIds: blockedUserIds);
  }
```